config/routes.rb

```diff
@@ -4,12 +4,16 @@
 
   get '/netflix_shows', to: "homes#index"
   get '/netflix_shows/:id', to: "homes#index"
+  get '/netflix_shows/:id/add_review', to: "homes#index"
```

Comment:
Since the form is on the show page and the path `netflix_shows/:id/add_review` is not something we are handling in `App.js` we can remove line 7.  All we need is line 6 currently

Reply:
Removed line 7